From 8a327b0ab1ba633467edf147d01b00029dd0021c Mon Sep 17 00:00:00 2001
From: Wenlin Kang <wenlin.kang@windriver.com>
Date: Mon, 29 Feb 2016 17:05:40 +0800
Subject: [PATCH] tmpfiles.d: don't create the symbol link to
 /usr/lib/os-release

The os-release is shipped by base-files in other distributions.
Recently, the base-files changed the os-release directory from
/etc/os-release to /usr/lib/os-release, which causes systemd boot
up failed, so a fix was added into systemd is

ln -s /usr/lib/os-release /etc/os-release

However, the os-release file shipped by os-release in Yocto project.
The location is still /etc/os-release, so systemd doesn't create the
symbol link.

Upstream-Status: Inappropriate [Yocto specific]

Signed-off-by: Yue Tao <yue.tao@windriver.com>
Signed-off-by: Wenlin Kang <wenlin.kang@windriver.com>
---
 tmpfiles.d/etc.conf.m4 | 1 -
 1 file changed, 1 deletion(-)

diff --git a/tmpfiles.d/etc.conf.m4 b/tmpfiles.d/etc.conf.m4
index e74b026..8e39eb2 100644
--- a/tmpfiles.d/etc.conf.m4
+++ b/tmpfiles.d/etc.conf.m4
@@ -7,7 +7,6 @@
 
 # See tmpfiles.d(5) for details
 
-L /etc/os-release - - - - ../usr/lib/os-release
 L /etc/localtime - - - - ../usr/share/zoneinfo/UTC
 L+ /etc/mtab - - - - ../proc/self/mounts
 m4_ifdef(`ENABLE_RESOLVED',
-- 
1.9.1

